lists/list_cache_test.go

```diff
@@ -131,7 +131,7 @@ var _ = Describe("ListCache", func() {
 			})
 		})
 		When("non transient err occurs on download", func() {
-			It("should delete existing elements from group cache", func() {
+			It("should keep existing elements from group cache", func() {
```

Comment:
see comment above

Reply:
I replied on the other comment too.

This test has an error occur on second parse, so it's not the case you mentioned (no entries and no error).  
I'll add a test for that case.

Reply:
I replied on the other comment too.

This test has an error occur on second parse, so it's not the case you mentioned (no entries and no error).  
I'll add a test for that case.

Reply:
Added a test with https://github.com/0xERR0R/blocky/pull/884/commits/08b17e2cd99f845bbb5231e9a293598f619e30d6